app/views/layouts/_flash_messages.html.slim

```diff
@@ -0,0 +1,4 @@
+.info-block
+  - flash.each do |key, value|
+    div[class = "alert alert-#{key == "notice" ? "info" : "warning"}"]
```

Comment:
If you need this key transformation for Bootstrap styles then you can try to do it from css side like suggested here - https://github.com/plataformatec/devise/issues/2051#issuecomment-254652729